formatTest/unit_tests/expected_output/jsx_functor.re

```diff
@@ -0,0 +1,17 @@
+type t = {displayName: string};
+
+let div = (~children=[], ()) => {
+  displayName: "div",
+};
+
+module X = {
+  let createElement = (~children=[], ()) => {
+    displayName: "X",
+  };
+};
+
+module F = (X: (module type of X)) => {
+  let createElement = X.createElement;
+};
+
+let body = <div> <F(X) /> </div>;
```

Comment:
Are several arguments supported?
```reason
<F(X,Y) />
```

Reply:
I've tried to add a test for `<F(X,Y) />` but I'm getting a syntax error on the file generated in `actual_output`. Could you try adding a multi argument example to the tests?

Reply:
Hm, I think the syntax error is from the idempotent tests. When I test it with `refmt` it doesn't seem to print right. Looking into this now